### PR TITLE
kubernetes client and other dependies are updated

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

- [x] Kubernetes client dependencies are updated to v1.25
- [x] FsInfo method of util package has been replaced with `Info` method
- [x] Most of the dependencies are updated to latest   

> /kind cleanup

```release-note
This release updates kubernetes client and other dependencies to latest
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,74 +1,99 @@
...
+	k8s.io/component-base v0.25.2 // indirect
+	k8s.io/mount-utils v0.25.2 // indirect
+)
+
+replace k8s.io/api => k8s.io/api v0.25.0
+
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.25.0
 
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.0
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.26.0-alpha.0
```
Comment (xing-yang) [inline go.mod:51]: Why v0.26.0-alpha.0?  Shouldn't we use the same version through out this file?

File: go.mod [hunk 2]
```diff
@@ -1,74 +1,99 @@
...
 
-replace k8s.io/apimachinery => k8s.io/apimachinery v0.21.0-alpha.0
+replace k8s.io/apiserver => k8s.io/apiserver v0.25.0
```
Comment (xing-yang) [inline go.mod:53]: Why not v0.25.2?

**Comment by humblec** [comment]
@xing-yang addressed the comments.. ptal . thanks!

**Pull request merged**